Application.js

```diff
@@ -57,22 +72,23 @@ define(["require", "dcl/dcl", "decor/Stateful", "decor/Evented", "dojo/Deferred"
 				// controllers: Array
 				// 		controller configuration array.
 				// returns:
-				// 		controllerDeferred object
+				// 		controllerPromise
```

Comment:
Yes, and I will change this function to have it always return a promise, and resolve it immediately if there are no controllers listed.